Lab2/BLL/Interfaces/IService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace BLL.Interfaces
+{
+    public interface IService<T>
+    {
+        Task Create(T item);
+
+        Task Delete(int id);
+
+        Task Update(T item);
+
+        Task<T> GetById(int id);
+
+        Task<IEnumerable<T>> GetAll();
```

Comment:
Return List or Array instead of IEnumerable.
FIx in all methods.

Reply:
Fixed